mp_compile.py

```diff
@@ -77,4 +77,5 @@ def install():
               "%s processes" % MAX_PROCS)
 
 
-install()
+if __name__ == "__main__":
+    install()
```

Comment:
The whole point of this module is that it monkeypatches the installer when it's imported. It can't run as a separate script, so the main clause isn't appropriate.